### DISCUSSION TITLE
Please correct your false marketing.

### DISCUSSION THREAD

**Discussion by i5hi** [btcpayserver/btcpayserver#2612] [General]
Allow me to share some background to explain exactly why the false marketing of BTCPayServer is dangerous to our community.

I am from India, a country where most of the population is under-educated. For a country like India to adopt Bitcoin, education is the prerequisite. Now, when it comes to legal tender and money, India understands a single `unit` of account, like Gold. This makes the process of initiation onto Bitcoin easier. 

Now, we have been looking for solutions to provide vendors - from cafes, restaurant, chaiwalas and paan walas in India. Your marketing led us to you. And then the moment we tried the software all these shitcoins emerged at the root! 

A few problems here:

1. You lied in calling it BTCPayServer; where it should be ShitcoinPayServer
2. Vendors will now be confused and ask about these shitcoins and want shitcoin support too.
3. The educated among us, whose duty it is to pave the way for Bitcoin as legal tender and raise the standard of living, will also think it is okay to equate shitcoins to Bitcoin in the payments environment (this is possibly a bigger issue because educated people blindly follow this project by virtue of appeal to authority)

Shitcoining in India is a problem and this project is worsening it. It is making our work on Bitcoin more difficult.

This is bad for our community. Please either change your name OR stop supporting shitcoins as a payment processor with BTC leading its name. Anyway, these shitcoins do not stand a chance to compete with Bitcoin as legal tender. Put more work into supporting lightning and real BTC.

This project seems to have casually become the evil it set out to fight.

You can correct this and we hope you do!

**Comment by Kukks** [comment]
You've clearly not attempted to use btcpay. We only develop and work on bitcoin related features, but we allow other communities to merge their altcoin support upstream is they do the work. We do not provide support or a place on the roadmap for anything other than bitcoin features.

Further to that, you can deploy btcpay in bitcoin only mode (and is the default if you do not explicitly attempt to add shitcoins) where not even any of the non bitcoin code makes it to the deployed application.

So get off your fucking high horse. This is free open source self hosted software under an MIT license that lets you do literally anything you want with it. If there are communities that want to add support for their shitcoin which doesn't complicate our architecture, they're free to do so as they also end up providing optimisations and new features and bug fixes along the way (as has been the case with dozens and dozens of contributions over the past few years).

**Comment by Zaxounette** [comment]
Hey @vmenond !

First of all, it would have been appreciated that you don't open an **issue**, but stick to a **discussion** since issues are for bug reports, as **stated in the obnoxious pre-formattted message that you deleted and discarded** before you started typing your message.

As for your actual message content here is, briefly, how the BTCPayServer project works:
* The BTCPayServer foundation develops and supports only Bitcoin (and associated L2, ie Lightning Network)
* All Altcoin integrations are supported & maintained by their respective communities

This lead us a few months ago to separate the project into two different builds.
Upon deployment, you now have the choice between the default Bitcoin-only build, and an Altcoin supporting build.
If you deploy the Bitcoin-only build, you will not see or interact with the Altcoin code in any manner.

**Comment by i5hi** [comment]
Man, really? Youre gna give me these excuses. I know all this. I have tried it. 

If you dont get it, i will not waste any more time.

**Comment by i5hi** [comment]
Its called BTCPayServer. Why not push the shitcoining to an entirely separate project. Do you guys not see the importance in that?

And no need to get abusive @Kukks , chill. I suppose you deep down hate youself for being involved in a sneaky shitcoining project; whatever it is, you, as a custodian of this project ought to be more civil.

**Comment by i5hi** [comment]
I didn't come here to fight. So I will close this issue. I have stated my peace. I hope you guys do the right thing.

**Comment by Kukks** [comment]
> I didn't come here to fight. So I will close this issue. I have stated my peace. I hope you guys do the right thing.

You clearly did come here with the intention to aggravate and instigate with passive aggressive comments. I gave you our reasoning and you shrugged them off as "excuses" and called the project "evil". If you can't see the benefit and value of this project beyond some opt in insignificant features (that we do not market, encourage and support), that's on you.

**Comment by i5hi** [comment]
Okay man, Im frustrated and doing my best at being civil. This is a serious problem to me and as project leaders you guys must be able to handle this better.

**Comment by i5hi** [comment]
Your 'reasoning' is already in the README - I get it, but its just an excuse - to be okay with shitcoining. Thats what i was getting at. Im not just here to rant man, i tried your software and i was let down.

**Comment by Kukks** [comment]
> Your 'reasoning' is already in the README - I get it, but its just an excuse - to be okay with shitcoining. Thats what i was getting at. Im not just here to rant man, i tried your software and i was let down.

The only way you were let down by shitcoins by testing it is if you actively installed shitcoins on your instance..

**Comment by i5hi** [comment]
The setup asks you to chose which shitcoins you want to activate...

**Comment by Eskyee** [comment]
I understand the feeling, shitcoins in BTCPay have put me off too, truth be told.. keepin it honest to bitcoin should of been our long term goal. 
I don't understand how we ever mixed a bitcoin project with a Shitcoin project.
my opinion is known to my team. And is my own opinion... also many people don't understand. Bitcoin let alone Shitcoin.. Esky rant over

**Comment by i5hi** [comment]
Look man, this is besides the point. I get that you have separated shitcoins from bitcoin and what not - thats the bare minimum you had to do. If you dont see the misleading nature of a project called BTCPayServer that supports shitcoins, there is not much more for me to say. From where I stand, this is doing harm because people look up to this project, and when it supports shitcoins casually, they think these are legitimate alternatives for legal tender and makes the entire process of what Bitcoin started more difficult and it makes it more difficult for the few of us who are keeping it Bitcoin-only.

**Comment by i5hi** [comment]
@Eskyee Thank you man!

**Comment by britttttk** [comment]
Fork the project. This conversation sounds familiar! @vmenond

**Comment by i5hi** [comment]
@britttttk thats too much work, why not present reason and engage in discussions and improve what exists? There are other projects out there like cyphernode and umbrel that are keeping it real to Bitcoin and could use support. That's where I have redirected my efforts. 

The people behind this project are intelligent and influential. At the very least this is important feedback for them.

**Comment by i5hi** [comment]
You are delaying the Bitcoin Standard.

**Comment by i5hi** [comment]
@NicolasDorier please consider this.

**Comment by pavlenex** [comment]
Ok, you said what you had, thanks for your feedback, please don't use issues for discussing something. We use issues to keep a track of bugs/to-do. You're free to continue this discussion in `discussions` or on our chat. 

You can read more about our bitcoin-only build here https://blog.btcpayserver.org/btcpay-server-1-0-5-4/
And in our documentation.

**Comment by i5hi** [comment]
Sorry for starting this discussion in the issues. 

Please read past my arrogance and hear the sentiment. I genuinely felt let down by this project  - this is why I just barged in. 

Although I am close to, and interact with, those who are far less fortunate than us on a daily basis, I still lack the understanding of the true nature of the problem that unsound money creates in poverty and the lowest class. I am in a very privileged position in India, and a lot of you much more than myself. It gets harder and harder the further up you are to connect to the core of this issue. It is something I seek to learn about more everyday and something Bitcoin has empowered me to do. What I do understand is that Bitcoin gives these people (and the rest of us) hope ; and shitcoins only add sophistication and muddy the waters for our understanding of this very complex systemic problem of an unsound monetary base. There are a few 'blockchain' companies in India but none focus on building Bitcoin infrastructure towards creating a more sound monetary base. Its still just the rich getting richer. Its still a shitcoin blockchain business model.  

**You folks have built a very respectable project and the decisions you make set the stage for developing countries like India because people here look up to projects like this.** 

When young new developers enter this space and use BTCPayServer and see all the shitcoin options, they get distracted by the noise and lose the signal. When existing projects use it, their shitcoining gets validated.

BTCPayServer should focus on being Bitcoin-only and not support any Shitcoin plug in natively. That belongs as an alternative project like XShitcoinPayServer which is a fork of BTCPayServer. This project must be Bitcoin-only. Encouraging shitcoining under a BTC title can not be considered okay. It is misleading. All the more so in the context of payments, which for most countries requires the asset in question to have the status of legal tender. With all the reckless shitcoining happening here, we are pushing back a bitcoin standard by years. 

**What benefit is this project getting from supporting shitcoins as a payment server?** 


Please do not respond back with another link to the bitcoin-only build. Or an explaination on how you can run BTCPay without any shitcoin traces. Ill just take those as a 'not interested' which is fine - just means I misunderstood the vision of this project.

**Reply by i5hi** [reply]
Leave no room for shitcoining. In the payment context especially. Bitcoin and Lightning is all we need. More signal, less noise.

**Reply by Eskyee** [reply]
All feedback is important to BTCPAY,  please bring your discussion here. https://chat.btcpayserver.org as everyone has a equal voice.. and I know BTCPAY core value is bitcoin and the developers are very dedicated to bitcoin.. our issues here are really for bug finding & fixes, plus new feature requests... 
please join us at Mattermost..
Keep your NRG for India 🇮🇳 & be lucky xx Peace

**Comment by britttttk** [comment]
>More signal, less noise

Take your own advice and make a PR or fork the project. 

>What benefit is this project getting from supporting shitcoins as a payment server?

Already answered by kukks

**Reply by i5hi** [reply]
@britttttk What is it with you and going around encouraging forking? That is a given when communication breaks down. We should be trying to solve the problems though discourse rather than encouraging forks  -_-

**Reply by britttttk** [reply]
> We should be trying to solve the problems though discourse rather than encouraging forks

That is the path of least resistance for your relentless demands that appear to reiterate the same information over and over.

**Reply by Kukks** [reply]
Forking of btcpay is actively encouraged. We do not want ourselves to be a systemic point of failure (eg you think we are evil and our values are corrupt, so fork the project if you think you can do better) and active forks provide parallel innovations which could always find its way upstream.

**Reply by i5hi** [reply]
@Kukks Man come one. At this stage can you please just see past my frustration. This is a real problem. You might not be able to relate sitting where you are. Come live with me here in India for a few days and try to understand our situation and tell me that I am not wrong for being a bit ticked off! 

A bitcoiner who wants more focus is asked to fork. But shitcoiners ask for support and you build solutions for them?

**Reply by i5hi** [reply]
And this is not just the case for India, this is every developing country. They need us to be focussed for them. These are simple people.

**Reply by i5hi** [reply]
@britttttk and you think there is no basis to these demands. These are real concerns that you should consider. I am just a regular guy who cares about raising the standard of living. There are many more like me. We are ready to run and do what it takes on the ground to bring this vision to life, but we need support from projects like BTCPayServer to stay true and build us the solutions we require.

**Reply by i5hi** [reply]
Please forgive me if i was hurtful, dont let my ego get in the way of a much larger issue. But the fact will remain that if nothing is done about this, then everything I said would sadly hold true.

**Reply by Zaxounette** [reply]
> A bitcoiner who wants more focus is asked to fork. But shitcoiners ask for support and you build solutions for them?
 
 That's probably the point of confusion right there.

The BTCPay Server Foundation doesn't touch altcoins/shitcoins.
Integrations other than Bitcoin are developed, maintained and reviewed by their respective communities. We don't build solutions for them. We don't review the code that enables shitcoin integration. If an integration doesn't work, isn't maintained, that's on the shitcoin community to fix/update.
If unmaintained for a long enough time, the integration will be considered depreciated and the activation string will be simply removed from the altcoin-build.

The BTCPayServer Foundation's purpose is Bitcoin-only. Any integration on top is just people piggy-backing on an open-source software.

**Reply by i5hi** [reply]
@Zaxounette yes this seems to be the point of confusion from both sides. I understand that there is Bitcoin-only build and what not and no shitcoin code touches this code base etc. Thats all technical explainations. The fact of supporting shitcoins as a project instead of recognizing them as obstacles that slow down a Bitcoin standard is more of a social issue. You are giving me a technical explaination to a primarily social issue. This project should be AGAINST shitcoins in the payment space. Just say no to the shitcoiner and ask them to fork. Why does your setup ask to choose coins? Why is all this support provided on BTCPayServers website. This is what I am getting at. THis is where I see the problem and it misleads new comers and validates shitcoiners.

**Reply by i5hi** [reply]
I want to run with BTCPayServer in India and this is the only thing stopping me, because it leaves a bad taste in the mouth

**Reply by i5hi** [reply]
With BTCPayServer focussed on the Bitcoin vision I can start working TODAY. With alternatives I have to still do a lot more work in either forking or helping other projects get up to speed.

**Reply by i5hi** [reply]
I honestly only see this project gaining more respect by making such a move. Clear all traces of shitcoinery and stop allowing it. Recognize it as a hurdle and focus on a world where we have standardized Bitcoin and LN payment systems. We dont need shitcoins in the payment space and the core of this project KNOWS THIS!

**Reply by i5hi** [reply]
We can leave this at this. We get each others positions. I would just like to apologize if my initial response was harsh, please understand that it comes from a place of concern and this has been a learning experience for me too. 

All the best.
